support-frontend/assets/__mocks__/stripeMock.tsx

```diff
@@ -67,3 +67,5 @@ jest.mock('@stripe/react-stripe-js', () => {
 		useElements: mockElements,
 	};
 });
+
+export {};
```

Comment:
This file is just imported for its side effects, so doesn't actually need to export anything